### PR TITLE
events.py: remove unused Pause class

### PR DESCRIPTION
**Pull request by disconnect3d**

### CODE REVIEW

File: pwndbg/gdblib/events.py [hunk 1]
```diff
@@ -12,8 +12,7 @@
 
 import pwndbg.config
 
-debug = pwndbg.config.Parameter("debug-events", False, "display internal event debugging info")
-pause = 0
+debug = pwndbg.config.Parameter("debug-events", True, "display internal event debugging info")
```
Comment (gsingh93) [inline pwndbg/gdblib/events.py]: You probably don't want to make the default `True`?

Comment (disconnect3d) [reply pwndbg/gdblib/events.py]: No, good catch; that was from other testing

Comment (disconnect3d) [inline pwndbg/gdblib/events.py]:
```suggestion
debug = pwndbg.config.Parameter("debug-events", False, "display internal event debugging info")
```

**Comment by gsingh93** [comment]
This seems to be used in `StartEvent`.

**Comment by disconnect3d** [comment]
@gsingh93 how? there is an `if pause:` condition in `connect` I missed, but I don't think this is used in `StartEvent`? Mainly, I don't see the `Pause` class to be used anywhere.

**Pull request merged**